corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/content.less

```diff
@@ -43,4 +43,26 @@ body {
     background: white;
     margin-bottom: 20px;
   }
+}
+
+@media print{
+  * {
+    visibility: collapse;
+  }
+
+  .print-container, .print-container * {
+    visibility: visible;
+  }
+
+  .noprint-sub-container{
```

Comment:
This could just be `#submit-button` since that's the only place you're using it. I'm not positive that would be better, just musing.

Reply:
My line of thinking is if this ever needs to be expanded for new content that needs to be displayed on print, "print container" would be the general class to use and "noprint-sub-container" would be the general paired use.

But not sure if that is a practical way of doing it, what do you think?

Reply:
I think that's a reasonable argument, happy to leave the code as you have it.